benchmarking-cli/src/main.rs

```diff
@@ -0,0 +1,157 @@
+use clap::Parser;
+use miden::{Assembler, ProgramInputs, ProofOptions};
+use std::fs;
+use std::time::Instant;
+
+#[derive(serde::Deserialize, serde::Serialize)]
+pub struct InputFile {
+    pub stack_init: Vec<String>,
+    pub advice_tape: Option<Vec<String>>,
+}
+
+// Creates a `ProgramInputs` struct for Miden VM from the specified inputs.
+fn get_program_inputs(stack_init: &[u64], advice_tape: &[u64]) -> ProgramInputs {
+    ProgramInputs::new(stack_init, advice_tape, Vec::new()).unwrap()
+}
```

Comment:
Comment for the future: we should allow other types of advice inputs as well. We'll want to make the same update to the playground as well.